ai-machine-learning/thoth-s2i-provenance/deploy.md

```diff
@@ -0,0 +1,41 @@
+Now that you are familiar with the configurations our s2i image offers, let's try deploying our sample app using Thoth's S2I build process. 
+
+Assuming you have followed the steps from before are logged in as an admin into `myproject` in the openshift cluster lets deploy - 
+
+If you haven't forked the repo and made any changes, you can try deploy our version using - 
```

Comment:
I think we can safely say users follow the tutorial and do not fork the repo (not to confuse them).

Reply:
Thanks, changed it